src/warnet/version.py

```diff
@@ -0,0 +1 @@
+VERSION = "0.9.11"
```

Comment:
e75b02d1a8bee542ba54a0ff10480e22e45f0b84

Seems like overkill to me to have the version string required in two places and compare them all the time. Is `pyproject.toml` always bundled with the rest of the code? Can't we just read the version directly from the toml file? We could add `warcli version` to the cli?